code/parse/sexp.cpp

```diff
@@ -20775,14 +20777,20 @@ void ship_copy_damage(ship *target_shipp, ship *source_shipp)
 	}
 }
 
-void sexp_replace_texture(int n)
+void sexp_replace_texture(int n, bool skybox)
 {
 	auto old_name = CTEXT(n);
 	n = CDR(n);
 
 	auto new_name = CTEXT(n);
 	n = CDR(n);
 
+	if (skybox) {
```

Comment:
This should check that `Nmodel_instance_num` is >= 0 (it could be -1 if the mission does not have a skybox).